packages/types-dev/objects.d.ts

```diff
@@ -376,6 +376,9 @@ declare global {
 
             // Make it possible to narrow the object type using the custom property
             custom?: undefined;
+
+            /** disable sentry reporting */
+            disableDataReporting?: boolean;
```

Comment:
We have a state for this to control plugins active state as already discussed, see also https://github.com/ioBroker/ioBroker.admin/issues/2409

Reply:
disableDataReporting removed